CHANGELOG.md

```diff
@@ -36,32 +39,32 @@ Version 3.0.0 is an effort to make BookReader more modular, extensible, and easi
 Changes include:
 
 - Make BookReader easier to use, by adding `options` to the constructor, and adding new `options.data` option. The old way of overriding properties should still work, but it is deprecated. With `options.data`, all BookReader needs is the image URLs and dimensions. To have dynamic image URLs (eg for scaling), omit the URL from `options.data`, and include `options.getPageURI`.
-- Factor out extra features into plugins. See `plugins` directory. Example plugins include:  
-    - plugins.chapters.js - render chapter markers  
-    - plugins.search.js - add search ui, and callbacks  
-    - plugins.tts.js - add tts (read aloud) ui, sound library, and callbacks  
-    - plugins.url.js - automatically updates the browser url  
-    - plugins.resume.js - uses cookies to remember the current page  
-    - plugins.mobile_nav.js - adds responsive mobile nav to BookReader  
+- Factor out extra features into plugins. See `plugins` directory. Example plugins include:
+    - plugins.chapters.js - render chapter markers
```

Comment:
It looks like your editor removed the trailing spaces. I believe the extra spaces are necessary for Markdown, but maybe not for github-flavored Markdown.

Given that this would probably keep happening to unsuspecting developers in the future, I guess we should just see what happens.

Reply:
Wow, TIL about Markdown's trailing 2 spaces indicating a <br> break. I can't help thinking what a dangerous, non-intuitive, and invisible bit of formatting that is!

In this case, they're being used after list items, so removing them doesn't seem to cause any problems.

Reply:
I'm pretty sure it doesn't break because this is Github, but maybe I am mistaken. Either way, I am fine with this change.